### PR TITLE
Add drag and drop files

### PR DESCRIPTION
**Pull request by ostatni5**

### CODE REVIEW

**Comment by grzanka** [comment]
Try following test:
1. Open UI as it is on your branch
2. Open yaptide UI editor, and load a project from following JSON: https://gist.githubusercontent.com/grzanka/cd47cacce2f6d5986397630d76580d11/raw/e6ca6a23fbae78b04ddcfaf2582bc7c3f972e995/2022let_plan2.json
3. Go to "Input files" and click "Generate from editor". Several files will appear there, one of them will be `sobp.dat`.
4. Now, on Windows, I find some directory with 4 usual input files (beam.dat, geo.dat, detect.dat and mat.dat). I drag and drop these files into the yaptide UI.
5. I would expect that `sobp.dat` disappears, but thats not the case, it is still there.

**Comment by grzanka** [comment]
Another exercise:
1. Generate input files as in the comment above
2. Clear all the files
3. Move just one (mat.dat) files into drag and drop
4. Suddenly all the files all populated, despite the fact I've cleaned them
https://user-images.githubusercontent.com/7374733/230183049-4777c271-6c23-4e28-8173-553a0ee12ad3.mp4

**Comment by grzanka** [comment]
I am happy to stay with the situation that uploading new files is not removing the old files. But in such case I would like to see a good option to remove only part of the files or all of them. Cleaning seems not to work properly

**Review by grzanka — DISMISSED** [review]

**Comment by grzanka** [comment]
@Derstilon please review as well, to me it looks OK

File: src/services/ResponseTypes.ts [hunk 1]
```diff
@@ -20,12 +20,26 @@ export enum StatusState {
 export type Response = {
 	message: string;
 };
+export const OrderedInputFilesNames = [
+	'info.json',
+	'geo.dat',
+	'mat.dat',
+	'beam.dat',
+	'detect.dat',
+	'sobp.dat'
```
Comment (grzanka) [inline src/services/ResponseTypes.ts:29]: This is for a future issue: filename `sobp.dat` should not be hardcoded.
SHIELD-HIT12A "project" is a collection of any number of files with arbitrary names, user can specify in `beam.dat` that `USECBEAM plik.dat` and upload file named `plik.dat`. 

The only names which are "fixed" are 4 mandatory input files, rest is of any name.

Comment (Derstilon) [reply src/services/ResponseTypes.ts:29]: This is just for ordering and autocompletion purposes. It doesn't appear if the user won't provide it.

Comment (Derstilon) [reply src/services/ResponseTypes.ts:29]: If we ever need to mock input files in code to check something then IDE can suggest names that we might want to include
image
There is no difference for the users with or without sobp.dat filename

**Review by Derstilon — APPROVED** [review]

**Review by grzanka — APPROVED** [review]

**Pull request merged**